src/content/certifications/exam-faqs/terraform-associate-003.mdx

```diff
@@ -75,31 +92,31 @@ Review the [rules and policies](https://hashicorp-certifications.zendesk.com/hc/
 
 #### Renew by passing a professional-level exam
 
-**_Unexpired_ Terraform Associate 002 or 003 credentials:** 
+**_Unexpired_ Terraform Associate credentials:** 
 
-When you pass the Terraform Authoring and Operations Professional exam, you will receive the professional-level credentials (badge and corresponding certificate). You will also extend the expiration of your Terraform Associate 002 or 003 credentials. 
+When you pass the Terraform Authoring and Operations Professional exam, you will receive the professional-level credentials (badge and corresponding certificate). You will also extend the expiration of your Terraform Associate credentials. 
 
 #### Renew by passing an associate-level exam
 
-**_Unexpired_ Terraform Associate 002 credential:** 
+**_Unexpired_ Terraform Associate credentials:** 
 
-* You can take the Terraform Associate 003 exam starting 18 months after your previous exam date. 
+* You can take the Terraform Associate 003 or Terraform Associate 004 exam starting 6 months before your expiration date. 
 * You will receive a new, separate set of credentials that will reflect your recertification date. 
-* The expiration date of Terraform Associate 002 credentials will not be updated.
+* The expiration date of your existing Terraform Associate credentials will not be updated.
 
 **_Unexpired_ Terraform Associate 003 credential:** 
 
-* You can retake the Terraform Associate 003 exam starting 18 months after your previous exam date. 
+* You can retake the Terraform Associate 003 exam or the Terraform Associate 004 exam starting 6 months before your expiration date. 
 * The expiration date on your credentials will be extended.
```

Comment:
This section has incorrect information. What is written in Ashley's doc is correct for _once 004 is released_ but not correct for what is true today. Since what is here is not what is Ashley's doc I assume we were trying to modify it for today's state but missed the mark

- Someone will NOT get new, separate set of credentials if they pass the same exam again and there cert is unexpired.
- Someone's 003 unexpired credentials will NOT be extended if they pass 004